src/Symfony/Component/AssetMapper/Compressor/ZstandardCompressor.php

```diff
@@ -43,6 +43,9 @@ private function createStreamContext()
 
     private function compressWithBinary(string $path): void
     {
+        if (null === $this->executable) {
```

Comment:
all these here and above should be reverted, they're already checked in CompressorTrait

Reply:
Reverted, only added to make static analysis happy.

The proper way to fix these errors would be refactoring these methods so that `$executable` is passed in as a `string`.